h2o-py/h2o/astfun.py

```diff
@@ -7,15 +7,15 @@
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-from opcode import *  # an undocumented builtin module
+import dis
```

Comment:
using "public" `dis` module here to avoid depending on non-official modules that may not be available if not using `Cython` implementation.